README.md

```diff
@@ -1 +1,46 @@
-# truex-ios-mobile-reference-app
+# Overview
+
+This project contains sample source code that demonstrates how to integrate the true[X]
+Ad Renderer in iOS. This document will step through the various pieces of code that make
+the integration work, so that the same basic ideas can be replicated in a real production app.
+
+This reference app covers the essential work. It assumes your app already have a working ad manager.
+~~For a more detailed integration guide, please refer to: https://github.com/socialvibe/truex-mobile-integrations/~~ (OUTDATED)
+
+# Access the true[X] Ad Renderer Library
+
+One can get the true[X] Ad Renderer either by
+[CocoaPods](https://github.com/socialvibe/cocoapod-specs) 
+```
+source 'https://github.com/socialvibe/cocoapod-specs.git'
```

Comment:
May want to highlight that this is a non-standard CocoaPods integration in that we use our own spec repo. Maybe linking out to this page: https://guides.cocoapods.org/making/private-cocoapods.html